### PR TITLE
Small tutorial improvements

### PR DESCRIPTION
**Pull request by CTNicholas**

### CODE REVIEW

File: docs/pages/api-reference/liveblocks-react.mdx [hunk 1]
```diff
@@ -1059,10 +1064,10 @@ The received input to all selector functions is a **read-only** and
...
 For example, suppose you have set up Storage in the typical way by setting
 `initialStorage` in your [`RoomProvider`][] to a tree that describes your app’s
-data model using `LiveList`, `LiveObject`, and `LiveMap`.
-The "root" argument for your selector function, however, will receive **an
-immutable and read-only representation** of that Storage tree, consisting of
-"normal" JavaScript datastructures. This makes consumption much easier.
+data model using `LiveList`, `LiveObject`, and `LiveMap`. The "root" argument
+for your selector function, however, will receive **an immutable and read-only
+representation** of that Storage tree, consisting of "normal" JavaScript
+datastructures. This makes consumption much easier.
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-react.mdx:1070]: Just prettier

File: docs/pages/api-reference/liveblocks-react.mdx [hunk 2]
```diff
@@ -1111,28 +1116,26 @@ Selectors you write can return _any_ value. You can use it to “just” select
...
-should return two results that are _referentially equal_. Special care needs to
-be taken when filtering or mapping over arrays, or when returning object
-literals, because those operations create new array or object instances on every
-call. (Why this is is detailed
+💡 **Important** One important rule is that selector functions **must return a
+stable result** to be efficient. This means calling the same selector twice with
+the same argument should return two results that are _referentially equal_.
+Special care needs to be taken when filtering or mapping over arrays, or when
+returning object literals, because those operations create new array or object
+instances on every call. (Why this is is detailed
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-react.mdx:1124]: Just prettier

File: docs/pages/api-reference/liveblocks-react.mdx [hunk 3]
```diff
@@ -1111,28 +1116,26 @@ Selectors you write can return _any_ value. You can use it to “just” select
...
-  [`shallow`][] here.
+- ⚠️ `(root) => root.animals.map(...)` is **not** stable 👎 Because `.map()`
+  creates a new array instance every time. You’ll need to use [`shallow`][]
+  here.
 
-- ✅ `(root) => root.animals.map(...).join(', ')` is stable 👍
-  Because `.join()` ultimately returns a string and all primitive values are
-  always stable.
+- ✅ `(root) => root.animals.map(...).join(', ')` is stable 👍 Because `.join()`
+  ultimately returns a string and all primitive values are always stable.
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-react.mdx:1138]: Just prettier

File: docs/pages/api-reference/liveblocks-react.mdx [hunk 4]
```diff
@@ -212,10 +212,10 @@ collaborative experiences as you build them, in real-time.
 Return the presence of the current user, and a function to update it.
 Automatically subscribes to updates to the current user’s presence.
 
-Note that the `updateMyPresence` setter function is different from the setter
-function as returned by React’s `useState` hook. Unlike the normal React state
-setter, you can pass a partial presence object to `updateMyPresence`, and it
-will get merged.
+Note that the `updateMyPresence` setter function is different to the setter
```
Comment (nvie) [inline docs/pages/api-reference/liveblocks-react.mdx:215]: So Bri'ish! 🇬🇧

File: docs/pages/api-reference/liveblocks-react.mdx [hunk 5]
```diff
@@ -1111,28 +1116,26 @@ Selectors you write can return _any_ value. You can use it to “just” select
...
 [in the next section](#selectors-subscribe-to-updates).)
 
 Some examples of what "a stable result" means:
 
-- ✅ `(root) => root.animals` is stable 👍
-  Liveblocks guarantees this. All nodes in the Storage tree are stable
-  references as long as their contents don’t change.
+- ✅ `(root) => root.animals` is stable 👍 Liveblocks guarantees this. All nodes
+  in the Storage tree are stable references as long as their contents don’t
+  change.
```
Comment (nvie) [inline docs/pages/api-reference/liveblocks-react.mdx:1131]: I think these have been lost in translation here somewhere along the lines. I once deliberately put a newline after the "👍" using the Markdown syntax by ending that line with `  ` (two spaces) before the newline. Prettier _should_ respect that and keep the hard line break in there. Not sure if something changes in Prettier, or the spaces got removed at one point, or our MDX plugin doesn't respect those.

**Review by nvie — APPROVED** [review]
Good updates! 👌

File: docs/pages/api-reference/liveblocks-react.mdx (hunk 5 above)
Comment (CTNicholas) [reply docs/pages/api-reference/liveblocks-react.mdx]: Gotcha! It's still possible to do this with `<br />` but I've used some other elements for a better style.

**Pull request merged**